tests/common/include/tests/common/tools/intersectors/helix_cylinder_intersector.hpp

```diff
@@ -102,13 +100,14 @@ struct helix_cylinder_intersector {
 
         is.path = getter::norm(helix_pos);
         is.p3 = helix_pos;
-        constexpr local_frame local_converter{};
-        is.p2 = local_converter(trf, is.p3);
 
-        auto local3 = trf.point_to_local(is.p3);
-        // Explicitly check for radial match
-        is.status =
-            mask.template is_inside<local_frame, true>(local3, mask_tolerance);
```

Comment:
This sets the radial check to true, because I had convergence problems with the Newton algorithm at one point. I will check this again, but can we remove the template on the mask instead?

Reply:
Makes sense. I put it back